gbpservice/neutron/services/servicechain/drivers/oneconvergence_intercloud_driver.py

```diff
@@ -102,9 +106,24 @@ def _fetch_template_and_params(self, context, sc_instance,
         sc_instance_id = sc_instance['id']
         filters = {'name': [SVC_MGMT_PTG_NAME]}
         svc_mgmt_ptgs = self._grouppolicy_plugin.get_policy_target_groups(
-            context._plugin_context, filters)
+            admin_context, filters)
+
         pt_type = TRANSPARENT_PT
 
+        filters = {'id': [consumer_ptg_id]}
+        ptgs = self._grouppolicy_plugin.get_policy_target_groups(
+                    context._plugin_context, filters)
+        consumer_ptg_subnet_id = None
+        stitching_port_ip = None
+        if ptgs:
+            is_consumer_ptg_external = False
+            consumer_ptg_subnet_id = self._get_ptg_subnet(
+                                        context, consumer_ptg_id)
+        else:
+            is_consumer_ptg_external = True
+
+        insert_type = 'north_south' if is_consumer_ptg_external else 'east_west'
```

Comment:
Is there no explicit call to determine if external policy is confiured as consumer ptg?